README.md

```diff
@@ -31,7 +31,7 @@ This handler takes several arguments, two of which are required.
 
 ### Arguments:
 * `path` (string, required): The path **from the current working directory** (NOT the
-* project path) to the SAM Template.
+* project path) to the SAM Template. Paths to Jinja (.j2) files _are_ supported.
```

Comment:
i think this is one sentence?  remove the leading `*` would make more sense.  also would be nice to link `Jinja` to the `Jinja SAM Templates` section below.

Reply:
Good catch! Thanks! My IDE must have made the next bullet when I hit enter. I'll clean up this docs section.